scripts/functions/networks/networks.py

```
@@ -28,3 +28,8 @@ def migrate_v214_tapchangersteps(network_id, variant_num):
 def migrate_v221_limits(network_id, variant_num):
         response = requests.put(constant.MIGRATE_V221_LIMITS.format(networkId = network_id, variantNum = variant_num))
         response.raise_for_status()
+
+
+def migrate_v237_limits(network_id, variant_num):
+    response = requests.put(constant.MIGRATE_V237_LIMITS.format(networkId=network_id, variantNum=variant_num))
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)